website/docs/docs/quickstarts/dbt-cloud/redshift-qs.md

```diff
@@ -0,0 +1,285 @@
+
+5. Type `Redshift` in the search bar at the top and click **Amazon Redshift**.
+
+    <Lightbox src="/img/redshift_tutorial/images/go_to_redshift.png" title="Click on Redshift" />
+
+6. Confirm that your new Redshift cluster is listed in **Cluster overview**. Select your new cluster.
+
+<Lightbox src="/img/redshift_tutorial/images/cluster_overview.png" title="Available Redshift Cluster" />
+
+7. Click **Query Data**. You can choose the classic query editor or v2. We will be using the v2 version for the purpose of this guide.
+
+8. You might be asked to Configure account. For the purpose of this sandbox environment, we recommend selecting “Configure account”.
+
+9. Select your cluster from the list and fill out the credentials from the output of the stack:
+    - **Database** &mdash; `dbtworkshop`
+    - **User name** &mdash; `dbtadmin`
+    - **Password** &mdash; Choose a password and save it for later
+
+<Lightbox src="/img/redshift_tutorial/images/redshift_query_editor.png" title="Redshift Query Editor v2" />
+
+<Lightbox src="/img/redshift_tutorial/images/connect_to_redshift_cluster.png" title="Connect to Redshift Cluster" />
+
+10. Click **Create connection**.
+
+## Load data 
+
+Now we are going to load our sample data into the S3 bucket that our Cloudformation template created. S3 buckets are a cheap and simple way to store data outside of Redshift.
+
+1. The data used in this course is stored as CSVs in a public S3 bucket. You can use the following URLs to download these files. Download these to your computer to use in the following steps.
+    - [jaffle_shop_customers.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/jaffle_shop_customers.csv)
+    - [jaffle_shop_orders.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/jaffle_shop_orders.csv)
+    - [stripe_payments.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/stripe_payments.csv)
+
+2. Now we are going to use the S3 bucket that you created with CloudFormation and upload the files. Go to the search bar at the top and type in `S3` and click on S3. There will be sample data in the file already, feel free to ignore it or use it for other modeling exploration.
+
+<Lightbox src="/img/redshift_tutorial/images/go_to_s3.png" title="Go to S3" />
+
+
+3. Click on the `name of the bucket` S3 bucket.  If you have multiple S3 buckets, this will be the bucket that was listed under “Workshopbucket” on the Outputs page. The bucket will be prefixed with `dbt-data-lake`.
+
+<Lightbox src="/img/redshift_tutorial/images/s3_bucket.png" title="Go to your S3 Bucket" />
+
+4. Click **Upload**. Drag the three files into the UI and click the **Upload** button.
+
+<Lightbox src="/img/redshift_tutorial/images/upload_csv.png" title="Upload your CSVs" />
+
+5. Save the name of the S3 bucket. It should look like this: `s3://dbt-data-lake-xxxx`. You will need it for the next section.
```

Comment:
This portion is a little vague as there is no `save` button. Do they mean `upload`? There is an "upload status" page after this portion - is there anything that needs to be recorded from that page (it's something they can't come back to after they leave) other than noting a successful upload?